Specs/DateTime/German/DateTimeModel.json

```diff
@@ -599,7 +631,7 @@
             {
               "timex": "PT123.45S",
               "type": "duration",
-              "value": "123.45"
+              "value": "123,45"
```

Comment:
There is a known issue with representing the separator here. Currently it's inconsistent.
Ideally value should be a canonical form and there should be another field for the locally-formatted value.
We can check the current Dutch to see it's usage. This will be refactored later.